doc/classes/Input.xml

```diff
@@ -312,6 +312,106 @@
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="frequency" type="int" />
+			<param index="3" name="amplitudes" type="int[]" />
+			<description>
+				Starts "Multi Vibration" adaptive triggers effect on the specified joypad. This method will fail if the joypad is not DualSense.
+				Axis should be either [constant JOY_AXIS_TRIGGER_LEFT] for the left trigger or [constant JOY_AXIS_TRIGGER_RIGHT] for the right trigger.
+				[code skip-lint]frequency[/code] parameter can take values from 0 to 255 (inclusive).
+				[code skip-lint]amplitudes[/code] array can have at most 10 elements, each of them can take values from 0 to 8 (inclusive).
+				[b]Note:[/b] This feature is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="joy_adaptive_triggers_off">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<description>
+				Stops adaptive triggers effects on the specified joypad. This method will fail if the joypad is not DualSense.
+				Axis should be either [constant JOY_AXIS_TRIGGER_LEFT] for the left trigger or [constant JOY_AXIS_TRIGGER_RIGHT] for the right trigger.
+				[b]Note:[/b] This feature is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="joy_adaptive_triggers_slope_feedback">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="start_position" type="int" />
+			<param index="3" name="end_position" type="int" />
+			<param index="4" name="start_strength" type="int" />
+			<param index="5" name="end_strength" type="int" />
+			<description>
+				Starts "Slope Feedback" adaptive triggers effect on the specified joypad. This method will fail if the joypad is not DualSense.
+				Axis should be either [constant JOY_AXIS_TRIGGER_LEFT] for the left trigger or [constant JOY_AXIS_TRIGGER_RIGHT] for the right trigger.
+				[code skip-lint]start_position[/code] parameter can take values from 0 to [code skip-lint]end_position[/code] (inclusive).
+				[code skip-lint]end_position[/code] parameter can take values from [code skip-lint]start_position+1[/code] to 9 (inclusive).
+				[code skip-lint]start_strength[/code] parameter can take values from 1 to 8 (inclusive).
+				[code skip-lint]end_strength[/code] parameter can take values from 1 to 8 (inclusive).
+				[b]Note:[/b] This feature is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="joy_adaptive_triggers_vibration">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="position" type="int" />
+			<param index="3" name="amplitude" type="int" />
```

Comment:
Here, the `amplitude` argument comes before the `frequency` argument; however, in `joy_adaptive_triggers_multi_vibration`, `frequency` comes before `amplitudes`. Is there a particular reason for this ordering? If not, I think it'd be best for them to follow the same order.